saga-core/src/main/java/io/servicecomb/saga/core/CompositeSagaResponse.java

```diff
@@ -0,0 +1,27 @@
+package io.servicecomb.saga.core;
```

Comment:
Please add the license header for the new added file.

Reply:
done